Client/API/ApiRequestHandler.cs

```diff
@@ -0,0 +1,55 @@
+﻿using API;
+using System.Diagnostics;
+using System.Net.Http;
+using System.Net.Http.Json;
+
+
+namespace Client.API
+{
+    public static class ApiRequestHandler
+    {
+        private static string apiUrl = "https://localhost:7262";
+
+        private static readonly HttpClient _httpClient = new();
+        public static async Task<Country> FetchCountryOfTheDay()
+        {
+            try
+            {
+                return await _httpClient.GetFromJsonAsync<Country>($"{apiUrl}/Country/GetCountryOfTheDay");
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine(e);
+                throw new Exception($"Api call failed, {e}");
+            }
+        }
+
+        public static async Task<Country> FetchCountry(int id)
```

Comment:
It should be a string with the country ISO code to the database. The endpoint is not up to date with that yet, but should also be changed